src/utils/__tests__/loadable.test.tsx

```diff
@@ -42,10 +42,9 @@ describe('loadable', () => {
   })
```

Comment:
Oddly this works:
```tsx
  it('should render fallback if given one', async () => {
    const component = render(<LazyComponentWithFallback />)
    expect(await screen.findByText(/lazy-loaded/)).toBeInTheDocument()
    expect(component.container.firstChild).toMatchSnapshot()
  })
```
but this complains `Warning: A suspended resource finished loading inside a test, but the event was not wrapped in act(...).`:
```tsx
  it('should render fallback if given one', async () => {
    const component = render(<LazyComponentWithFallback />)
    expect(await screen.findByText(/Loading/)).toBeInTheDocument()
    expect(component.container.firstChild).toMatchSnapshot()
    expect(await screen.findByText(/lazy-loaded/)).toBeInTheDocument()
    expect(component.container.firstChild).toMatchSnapshot()
  })
```